index.html

```diff
@@ -2036,21 +2036,21 @@ <h4>Presentations Including Holder Claims</h4>
 A <a>holder</a> MAY use the <code>verifiableCredential</code> <a>property</a> in
 a <a>verifiable presentation</a> to include <a>verifiable credentials</a> from
 any <a>issuer</a>, including themselves. When the <a>issuer</a> of a
-<a>verifiable credential</a> is the <a>holder</a>, its <a>claims</a> are
-considered to be self-asserted. These self-asserted claims can be secured by
-their own mechanism like any other <a>verifiable credential</a> or by the same
-mechanism that secures the <a>verifiable presentation</a> in which they are
-included.
+<a>verifiable credential</a> is the <a>holder</a>, the <a>claims</a> in that
+<a>verifiable credential</a> are considered to be <em>self-asserted</em>.
+Such self-asserted claims can be secured by the same mechanism that secures
+the <a>verifiable presentation</a> in which they are included or by any
+mechanism usable for other <a>verifiable credentials</a>.
           </p>
           <p>
-The <a>subject</a> of these self-asserted <a>claims</a> is not limited and can
-include statements about the <a>holder</a>, one of the other included
-<a>verifiable credentials</a>, or even the <a>verifiable presentation</a> in
-which the self-asserted <a>verifiable credential</a> is included. In each of
-these cases the <code>CredentialSubject</code> would have an <code>id</code>
-<a>property</a> that matches the <code>id</code> of the <a>subject</a>. This is
-no different than how it is done in <a>verifiable credentials</a> that are not
-self-asserted.
+The <a>subject(s)</a> of these self-asserted <a>claims</a> are not limited, so
```

Comment:
this is complicated enough that it really ought to have an example.

Reply:
added an example in [bef11ba](https://github.com/w3c/vc-data-model/pull/1186/commits/bef11bab1f6d8dc111f10d669b529b99a50451ea)

Reply:
imo this can be resolved.